step.sh

```diff
@@ -46,6 +46,19 @@ if [ "${script_input}" == 'Fastlane copy' ]; then
 elif [ "${script_input}" == 'Prep Slack message' ]; then
 	copyFastfile
 else
-	gem install hockeyver
-	ruby "${THIS_SCRIPT_DIR}/parse_project_settings.rb"
+	# Check if we have ci-version set correctly
+	if [ -z "${CI_VERSION}" ]; then
```

Comment:
This might need to be a macro at some point since it's used a couple of times but I guess its fine for now